src/components/pager/__internal__/pager-navigation-link.component.tsx

```diff
@@ -85,8 +88,11 @@ const PagerNavigationLink = ({
 
   const { text } = navLinkConfig[type];
 
+  const hideDisabledButtons = hideDisabledElements && disabled();
+
   return (
     <StyledPagerLink
+      hideDisabledButtons={hideDisabledButtons}
```

Comment:
I'd recommend inside of hiding the element via css, using the `hidden` attribute like so:
```suggestion
      hidden={hideDisabledButtons}
```
The reason for this is hiding the element via css will visually hide the element, but it will still be recognised by the browser, so the links could potentially still be readable by screen readers. Using `hidden` instead will prevent the browser from rendering it which should avoid this issue.